### PR TITLE
Fix remaining issues from PR #167, namely equation 105

### PR DESCRIPTION
**Pull request by Isweet**
This addresses remaining issues of PR #167.

I will close the other PR in favor of this one.

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -755,7 +755,7 @@ \section{Message Call} \label{ch:call}
 0 & \text{if} \quad \boldsymbol{\sigma}^{**} = \varnothing \\
 g^{**} & \text{otherwise}
 \end{cases} \\
-(\boldsymbol{\sigma}^{**}, g^{**}, \mathbf{s}, \mathbf{o}) & \equiv & \begin{cases}
+\qquad (\boldsymbol{\sigma}^{**}, g^{**}, A, \mathbf{o}) & \equiv & \begin{cases}
```
Comment (pirapira) [inline Paper.tex]: Does the `\qquad` make a difference here?  I thought the whole tuple is right-aligned toward the `\equiv` symbol.

Comment (Isweet) [inline Paper.tex]: The equation numbering overlaps with the equation. This just scoots things over a bit. If theres a more elegant solution than qquad i'd be happy to do that instead.

Comment (pirapira) [inline Paper.tex]: I see.  I have the same issue sometimes.  Thanks.

**Comment by pirapira** [comment]
Looks good.

**Pull request merged**